apps/explorer/src/app/routes/network-parameters/network-parameters.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
follow up (non-blockng): we need to be consistent with our file naming, some tests are .test.ts and some are .spec.ts, we should also ideally look into linting this 👍